packages/block-library/src/tag-cloud/block.json

```diff
@@ -6,6 +6,12 @@
 	"description": "A cloud of your most used tags.",
 	"textdomain": "default",
 	"attributes": {
+		"tagsToShow": {
```

Comment:
Yes! I considered this, and I like that name a bit better.

For context, I chose this name to match with `commentsToShow` from [Latest Comments](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/latest-comments/block.json#L10) and `postsToShow` from [Latest Posts](https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/latest-posts/block.json#L19), which follow a similar pattern.

I'm happy to change it -- just wanted to be sure the reasoning was documented, in case it makes any difference in decision-making.

Reply:
Thanks! I guess the reason was that Latest Posts is really old and had that naming and the person introduced Latest Comments copied the name 😆 

Your instinct to have a better label and fixture name was good `core__tag-cloud__numberOfTags.html` 😄

Reply:
Thanks again! I believe these concerns are addressed in the most recent commits.

I'll leave this open for you to resolve based on the review.